src/System.Numerics.Vectors/src/System/Numerics/Matrix4x4.cs

```diff
@@ -2346,9 +2181,9 @@ public bool Equals(Matrix4x4 other)
         /// <returns>True if the Object is equal to this matrix; False otherwise.</returns>
         public override bool Equals(object obj)
         {
-            if (obj is Matrix4x4)
+            if (obj is Matrix4x4 m)
```

Comment:
Nit: You could do this in one line with `(obj is Matrix4x4 other) && (this == other)`